Week1/Section3/Exercise7/Source7.cpp

```diff
@@ -19,44 +41,36 @@ int main()
 	/* initialize random seed: */
 	srand(time(NULL));
 
-	for (int i = 0; i < sizeof(Lottery) / sizeof(int); i++)
+	int LotteryArraySize = sizeof(Lottery) / sizeof(int);
+
+	for (int i = 0; i < LotteryArraySize; i++)
 	{
 		// populate the Lottery array
 		int RandomNumber = 1 + rand() % 20;
 		bool isRandomRepeated = false;
```

Comment:
Same here, remember the bool conventions. (I know this is old code that I didn't comment on in the previous review, but I am just realizing it 😅